lib/fastlane/plugin/ddg_apple_automation/helper/ddg_apple_automation_helper.rb

```diff
@@ -22,6 +22,12 @@ def self.asana_task_url(task_id)
       def self.path_for_asset_file(file)
         File.expand_path("../assets/#{file}", __dir__)
       end
+
+      def self.load_template_file(template_file)
+        File.read(template_file)
+      rescue StandardError
+        UI.user_error!("Error: The file '#{template_file}' does not exist.")
+      end
```

Comment:
Curious if we's ever need to call `path_for_asset_file` standalone. If not, we could perhaps combine these two functions int one and have a function that computes the path and reads the file. And then write tests for it :)

Reply:
makes perfect sense, but let me do it in another PR today!